www/include/views/graphs/exportData/ExportCSVServiceData.php

```diff
@@ -33,12 +33,7 @@
  *
  */
 
-function get_error($str)
-{
-    echo $str . "<br />";
-    exit(0);
-}
-
+require_once realpath(__DIR__ . "/../../../../../bootstrap.php");
```

Comment:
Why include this file and not just the one needed?

Reply:
@callapa that's true, i just thought maybe i included the bootstrap in case using more of the wired classes in the future ^^

Reply:
I understand. For this particular case, you just need to include the necessary file.
We will see how to include the autoloader next time.

Reply:
@callapa Done :) changes has been made !